btcec/signature.go

```diff
@@ -21,22 +20,17 @@ import (
 var (
 	errNegativeValue          = errors.New("value may be interpreted as negative")
 	errExcessivelyPaddedValue = errors.New("value is excessively padded")
+	one                       = big.NewInt(1)
 )
 
 // Signature is a type representing an ecdsa signature.
 type Signature struct {
-	R *big.Int
-	S *big.Int
+	R       *nfieldVal
+	S       *nfieldVal
+	SNegPad bool
```

Comment:
This is a test-specific thing and shouldn't be part of the struct that non-test code has to use.